src/process/process_set.cu

```diff
@@ -266,12 +325,26 @@ namespace micm
       if (devstruct.yields_ != nullptr)
         CHECK_CUDA_ERROR(
             cudaFreeAsync(devstruct.yields_, micm::cuda::CudaStreamSingleton::GetInstance().GetCudaStream(0)), "cudaFree");
+      if (devstruct.jacobian_process_info_ != nullptr)
+        CHECK_CUDA_ERROR(
+            cudaFreeAsync(devstruct.jacobian_process_info_, micm::cuda::CudaStreamSingleton::GetInstance().GetCudaStream(0)),
```

Comment:
can we save `micm::cuda::CudaStreamSingleton::GetInstance().GetCudaStream(0)` to a local variable and simplify the interface here?